packages/api/webpack.config.js

```diff
@@ -1,7 +1,22 @@
+import { createRequire } from 'module'
+import webpack from 'webpack'
+
+const require = createRequire(import.meta.url)
+
 export default {
   target: 'webworker',
   mode: 'development',
   devtool: 'cheap-module-source-map', // avoid "eval": Workers environment doesn't allow it
-  plugins: [],
+  plugins: [
+    new webpack.ProvidePlugin({
+      process: 'process/browser',
```

Comment:
Which thing is using `process` in the browser? My assumption is the new `api/scripts` files are intended to be run from node

Reply:
`@magic-sdk/admin`